### DISCUSSION TITLE
Since Years it WAS the best Downloader......now: disimproved to the worse !!

### DISCUSSION THREAD

**Discussion by AnalAndy** [aclap-dev/video-downloadhelper#648] [Help and Q&A]
### Your question

I allways could download any video-clip, now it fails already on the most simple clips😕😠  dumb down

### Addon & CoApp version and details

Windows 11 14H2,  VideoDownloadHelper 9.01.13 plus installed DownloadHelper CoApp

**Comment by paulrouget** [comment]
Which website fails? Can't help you without details.

Also, if you don't like the new version, you can just switch back to the old one like this: https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface
